boards/arm/sensortile_box_pro/sensortile_box_pro.dts

```diff
@@ -158,10 +158,28 @@ stm32_lp_tick_source: &lptim1 {
 };
 
 &spi1 {
-	pinctrl-0 = <&spi1_nss_pe12 &spi1_sck_pe13
-		     &spi1_miso_pe14 &spi1_mosi_pe15>;
+	pinctrl-0 = <&spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pa7>;
 	pinctrl-names = "default";
+	cs-gpios = <&gpioa 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
 	status = "okay";
+
+	bluenrg_lp_sensortile_pro: bluenrg_lp@0 {
```

Comment:
normally the nodelabel goes `_` and the node with `-` but you could also skip the nodelabel as well since  I don't think you need it, checked other `zephyr,bt-hci-spi` instances in the code base and they indeed don't set it.

How about just

`bluenrg-lp@0 {`

?

Reply:
Yes, you're right. I'll change it.